losol/eventuras
# Resource based authorization POC

To test this PR:

1. Logout and create a new registration for event with id `1` (example).
1. Login as admin and set the registration's type to `staff`.
1. Click on the link sent to the new registration.
1. Now browse to `/Events/Participants/POC/1`, the request will succeed, because this is the event the user registered as a staff.
1. Then browse to `/Events/Participants/POC/2`, the request will fail as expected.

---

The claims for the resource based authorization are currently being set on confirmation of the registration email. So the admin must change the registration type to staff before the confirmation email link is set.

This PR doesn't handle the different use-cases to keep the event related claims upto-date, since it's only a POC.


Closes #268
